.config/dotnet-tools.json

```diff
@@ -3,13 +3,13 @@
   "isRoot": true,
   "tools": {
     "paket": {
-      "version": "5.257.0",
+      "version": "5.258.1",
       "commands": [
         "paket"
       ]
     },
     "fake-cli": {
-      "version": "5.20.3",
+      "version": "5.20.4",
```

Comment:
When I update FAKE I usually delete `build.fsx.lock` and re-generate during the next build with fresh dependencies

Reply:
Done it.